website/docs/docs/deploy/source-freshness.md

```diff
@@ -12,24 +12,25 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](/doc
 
 ### Enabling source freshness snapshots
 
-First, make sure to configure your sources to [snapshot freshness information](/docs/build/sources#snapshotting-source-data-freshness).
+[`dbt build`](reference/commands/build) does _not_ include source freshness checks when it builds and tests resources in your DAG. As such, here's a common pattern for defining jobs:
```

Comment:
Are they choosing one or can they do all three? I only ask because  "here's a common pattern for defining jobs" doesn't really tell me whether I should pick one or all three? Maybe because the sentence is passive?

```suggestion
[`dbt build`](reference/commands/build) does _not_ include source freshness checks when building and testing resources in your DAG. Instead, you can use one of these common patterns for defining jobs:
```

Reply:
this is how it is right now in the docs but yes they can pick all three as a common pattern. love the suggestion, thank you!